src/styles/_catalog.scss

```diff
@@ -14,6 +14,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: color, 300ms;
+
+  &:hover {
+    transition: color, 300ms;
```

Comment:
you have the same code inside hover and above it, it only needs to be in 1 place